Foundation/NSXMLNodeOptions.swift

```diff
@@ -43,55 +43,55 @@
     @constant NSXMLDocumentIncludeContentTypeDeclaration Include a content type declaration for HTML or XHTML
 */
 
-#if arch(x86_64) || arch(arm64)
-public typealias NSXMLNodeOptionRawType = Int
+#if arch(x86_64) || arch(arm64) || arch(powerpc64le)
```

Comment:
Yes, I believe so.  There are no configuration statements listed for bit-width in the Swift Programming Language book.  I would love for that to exist.